Extension/src/abTesting.ts

```diff
@@ -73,11 +95,19 @@ export class ABTestSettings {
                 this.recursiveIncludesDefault.Value = util.isNumber(newSettings.recursiveIncludes) ? newSettings.recursiveIncludes : this.recursiveIncludesDefault.DefaultValue;
                 this.gotoDefIntelliSenseDefault.Value = util.isNumber(newSettings.gotoDefIntelliSense) ? newSettings.gotoDefIntelliSense : this.gotoDefIntelliSenseDefault.DefaultValue;
                 this.enhancedColorizationDefault.Value = util.isNumber(newSettings.enhancedColorization) ? newSettings.enhancedColorization : this.enhancedColorizationDefault.DefaultValue;
+                const newMap: Map<string, string> = new Map<string, string>();
+                if (newSettings.minimumVSCodeVersion) {
+                    for (const [extensionVersion, vscodeVersion] of Object.entries(newSettings.minimumVSCodeVersion)) {
+                        newMap.set(extensionVersion, <string> vscodeVersion);
+                    }
+                }
+                this.minimumVSCodeVersionDefault = newSettings.minimumVSCodeVersion ? newMap : this.minimumVSCodeVersionDefault;
```

Comment:
Set `this.minimumVSCodeVersionDefault` to newMap within if statement block of line 99 since it checks for  `newSettings.minimumVSCodeVersion`. So no need to do another check on line 104.